src/app/ui/copy-to-clipboard.js

```diff
@@ -12,7 +12,7 @@ var css = csjs`
   }
 `
 
-module.exports = function copyToClipboard (getContent, tip = 'Copy value to clipboard', icon = 'fa-copy') {
+module.exports = function copyToClipboard (getContent, tip = 'Copied value to clipboard', icon = 'fa-copy') {
```

Comment:
this is the title(tooltip) of the button. IMHO this should stah "Copy"

Reply:
Okay, then we should probably have a separate message for the success pop up.